src/MissingTableHelper.php

```diff
@@ -59,13 +59,21 @@ private static function fillMissingTableMapping(
                     $primaryKey[] = $item['mapping']['destination'];
                 }
             } elseif ($item['type'] === 'table') {
+                if (empty($item['parentKey'])) {
+                    $parentKeyParam = ['destination' => $name . '_pk'];
```

Comment:
Takze ked v tej child tabulke `parentKey` nie je nastaveny, tak sa pouzije "default" nazov parent sloupce + '_pk'. To je asi v pohode teda. Je tam na to nejaky test?

Reply:
jop, tutok https://github.com/keboola/generic-extractor/pull/117/files#diff-6b3b53e020ad3e45990775eb05699f72R544

Reply:
jo v ty dokumentaci je to nejakej renonc

Reply:
issue jsem zalozil